src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/relatedPartyTransactions/AC7803.scala

```diff
@@ -30,8 +30,8 @@ case class AC7803(value: Option[String]) extends CtBoxIdentifier(name = "Name of
 
     collectErrors(
       validateAsMandatory(this),
-      validateOptionalStringByLength("AC7803", this, 1, StandardCohoTextFieldLimit),
-      validateOptionalStringByRegex("AC7803", this, ValidCoHoCharacters)
+      validateStringMaxLength("AC7803", value.getOrElse(""), StandardCohoTextFieldLimit),
```

Comment:
I tried to change it as you suggested but it breaks the text validation fixtures in a way I cant (yet) work out how to fix. About 15 existing boxes use the same pattern (which is why I used it ....), validateOptionalStringByRegex doesnt work with the text validation fixture which is why I aligned with whats already done